shlr/zip/zlib/ChangeLog

```diff
@@ -662,15 +666,19 @@ Changes in 1.2.3.2 (3 September 2006)
 - Rig configure --shared to build both shared and static [Teredesai, Truta]
 - Remove zconf.in.h and instead create a new zlibdefs.h file
 - Fix contrib/minizip/unzip.c non-encrypted after encrypted [Vollant]
-- Add treebuild.xml (see http://treebuild.metux.de/) [Weigelt]
+- Add treebuild.xml (see https://treebuild.metux.de/) [Weigelt]
```

Comment:
this server only responds on http:// no https is there. can you write a script to test all the urls before replacing them?

Reply:
A script will be difficult. I think I will have to check it manually